tools/compress_assets.py

```
@@ -0,0 +1,140 @@
+    # Write header file
+    with open(OUTPUT_FILE, "w", encoding="utf-8") as f:
+        f.write(header_content)
+
+    print("\n" + "=" * 60)
+    print(f"✅ Generated: {OUTPUT_FILE}")
+
+    total_original = sum(s["original"] for s in stats)
+    total_compressed = sum(s["compressed"] for s in stats)
+    total_ratio = (1 - total_compressed / total_original) * 100 if total_original > 0 else 0
+
+    print(f"\n📊 Total compression:")
```

Ruff (0.15.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)